### PR TITLE
UI: Rename Pipeline Marketplace to Tekton Hub & Upload to Add

### PR DESCRIPTION
**Pull request by sm43**

### CODE REVIEW

**Comment by sm43** [comment]
Screenshot from 2020-02-27 14-44-11

File: frontend/src/components/main/App.tsx [hunk 1]
```diff
@@ -72,7 +72,7 @@ const App: React.FC<mainProps> = () => {
         '1.3em', marginRight:
          '-0.6em', color: 'white'}}>
         {/* {' '} */}
```
Comment (PuneetPunamiya) [inline frontend/src/components/main/App.tsx]: Can this comment be removed

**Comment by sthaha** [comment]
@SM43 could we please keep the font the same as the old 🙏 ?

**Review by sthaha — CHANGES_REQUESTED** [review]
- Could you please use the old font? 
- Squash the commits or `rebase --interactive` to merge commits?

**Comment by sm43** [comment]
main

**Pull request merged**